### ISSUE TITLE
[12.0] OpenGL love_PixelCoord doesn't seem to be working.

### ISSUE THREAD

**Issue by ettiernu** [love2d/love#1866]
Works fine still on Vulkan, something with the split uniforms perhaps?

**Comment by slime73** [comment]
Can you provide some code demonstrating the issue you're seeing?

**Comment by ettiernu** [comment]
```
 local shader

function love.load()
  shader = love.graphics.newShader([[
  uniform float startY;
  uniform float height;

  vec4 effect(vec4 color, Image maintex, vec2 tc, vec2 pc)
  {
    vec4 result = color;
    result.rgb = mix(vec3(1.0, 0.0, 0.0), vec3(0.0, 0.0, 1.0), ((pc.y / love_ScreenSize.y) - startY)/height);
    return result;
  }
 ]])
end

function love.draw()
  love.graphics.setShader()
  local renderer = love.graphics.getRendererInfo()
  love.graphics.print(renderer)
  local height = love.graphics.getHeight()
  love.graphics.rectangle("line", 18, 58, 404, 404)
  love.graphics.setShader(shader)
  shader:send("startY", 60 / height)
  shader:send("height", 400 / height)
  love.graphics.rectangle("fill", 20, 60, 400, 400)
end
```
![Screenshot 2022-11-25 152529](https://user-images.githubusercontent.com/13109835/204039644-2edff7d9-6021-48af-980c-400275f13b57.png)
![Screenshot 2022-11-25 152656](https://user-images.githubusercontent.com/13109835/204039645-495aa8c0-e8cd-41f8-a607-8da47bd5909b.png)

**Comment by MikuAuahDark** [comment]
Can confirm seeing same result as OP in 12.0

**Comment by slime73** [comment]
Fixed in [6599697](https://github.com/love2d/love/commit/659969783542a18325ab8794126f96e62e4c1d6a). Thanks for the test code!
